docs/content/security.md

```diff
@@ -133,21 +133,36 @@ must be provided on startup. The authorization policy must be structured as foll
 # system.authz as follows:
 package system.authz
 
-default allow = false  # Reject requests by default.
+default response = { "allow" : false }  # Reject requests by default.
 
-allow {
+response = { "allow" : true } {
   # Logic to authorize request goes here.
 }
 ```
 
 When OPA receives a request, it executes a query against the document defined
-`data.system.authz.allow`. The implementation of the policy may span multiple
+`data.system.authz.response`. The implementation of the policy may span multiple
 packages however it is recommended that administrators keep the policy under the
 `system` namespace.
 
 If the document produced by the ``allow`` rule is ``true``, the request is
 processed normally. If the document is undefined or **not** ``true``, the
-request is rejected immediately.
+request is rejected immediately. Optionally a reason can be given for denied requests as follows.
+
+```live:system_ns:module:read_only
+# Deniying a request with reason
+package system.authz
+
+# Reject requests with a reason.
+default response = { 
```

Comment:
Indentation looks wonky here -- Could you run the code through `opa fmt`, maybe? 😃

Reply:
Updated the security.md